input/resources/StructureDefinition-profile-patient.json

```diff
@@ -45,6 +45,216 @@
         "isModifier": false
       },
       {
+        "id": "Patient.identifier",
+        "path": "Patient.identifier",
+        "slicing": {
+          "discriminator": [
+            {
+              "type": "value",
+              "path": "system"
+            }
+          ],
+          "rules": "open"
+        }
+      },
+      {
+        "id": "Patient.identifier:SSN",
+        "path": "Patient.identifier",
+        "sliceName": "SSN",
+        "short": "SSN Identifier for this patient",
+        "definition": "SSN identifier for this patient."
+				  
+      },
+      {
+        "id": "Patient.identifier:SSN.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:SSN.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "SB"
+      },
+      {
+        "id": "Patient.identifier:SSN.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "SSN"

```

Comment:
I was reading through https://www.hl7.org/fhir/valueset-identifier-type.html, change SSN to be this:
type.coding.display = Social Beneficiary Identifier

Reply:
changed